test/evolutionaryAlgorithm/SimulationComponents/SurvivorSelector/impl/test_SurvivorSelector.py

```diff
@@ -1,23 +1,25 @@
 from src.evolutionaryAlgorithm.SimulationComponents.Individual.impl.FloatingPointIndividual import \
     FloatingPointIndividual
-from src.evolutionaryAlgorithm.SimulationComponents.SurviviorSelector.impl.RouletteSurvivorSelector import \
-    RouletteSurvivorSelector
+from src.evolutionaryAlgorithm.SimulationComponents.SurviviorSelector.impl.eliteSurvivor import eliteSurvivor
 
 
 def test_SurvivorSelector():
-    selector = RouletteSurvivorSelector()
+    selector = eliteSurvivor()
     fitness = [0.2, 0.4, 0.5, 0.9]
-    population = [FloatingPointIndividual([1, 1, 1]), FloatingPointIndividual([2, 2, 2]),
-                  FloatingPointIndividual([3, 3, 3]), FloatingPointIndividual([4, 4, 4])]
-    i = 0
-    for el in population:
-        el.setFitnessFunctionEvaluation(fitness[i])
-        i = i + 1
+    population = [FloatingPointIndividual([1, 1, 1]), FloatingPointIndividual([2, 2, 2])]
+    offspring = [FloatingPointIndividual([3, 3, 3]), FloatingPointIndividual([4, 4, 4])]
 
-    survivors = selector.selectSurvivor(population)
+    temp = [population[0].getRepresentation(), offspring[0].getRepresentation()]
+
+    population[0].setFitnessFunctionEvaluation(fitness[0])
+    population[1].setFitnessFunctionEvaluation(fitness[1])
+    offspring[0].setFitnessFunctionEvaluation(fitness[2])
+    offspring[1].setFitnessFunctionEvaluation(fitness[3])
+
+    survivors = selector.selectSurvivor(population, offspring)
     num = len(survivors)
 
     assert num == 2
 
     for el in survivors:
-        assert el in population
\ No newline at end of file
+        assert el.getRepresentation() in temp
```

Comment:
Ten test jest podejrzany i trzeba bedzie go napisac jeszcze raz ( tak zeby faktycznei sprawdzac poprawnosc dzialania selectora). Ale to moze potem, bo zrobilam duzy refactor api